Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.Events.cs

```diff
@@ -198,16 +200,52 @@ private void Menu_Loaded(object sender, RoutedEventArgs e)
                 _wrapPanel.Orientation = Orientation;
             }
 
+            Window.Current.CoreWindow.PointerMoved -= CoreWindow_PointerMoved;
             LostFocus -= Menu_LostFocus;
             LostFocus += Menu_LostFocus;
             Dispatcher.AcceleratorKeyActivated -= Dispatcher_AcceleratorKeyActivated;
             Window.Current.CoreWindow.KeyDown -= CoreWindow_KeyDown;
             Dispatcher.AcceleratorKeyActivated += Dispatcher_AcceleratorKeyActivated;
             Window.Current.CoreWindow.KeyDown += CoreWindow_KeyDown;
+            Window.Current.CoreWindow.PointerMoved += CoreWindow_PointerMoved;
+        }
+
+        private void Menu_LayoutUpdated(object sender, object e)
```

Comment:
That's needs a bit of testing because LayoutUpdated will only trigger if any property is changed on the control and the menu is not expected to get updated frequently while on menu open that is really expected to be triggered a lot more than a change in any property :)